docs: enable section indexes (#11371)
PrairieLearn/PrairieLearn: docs/getting-started/index.md

# Getting Started

Welcome to PrairieLearn! Check out the following pages to get started:

- [Request a course space](../requestCourse/index.md): get access to your own course on PrairieLearn.

- [Fundamental concepts](../concepts/index.md): gain a high-level understanding of how PrairieLearn works.

- [Quick start](../getStarted.md): learn how to use and navigate the PrairieLearn interface.

> Afterward, you may want to check out the [instructor guide](../instructor-guide/index.md) for more details on how to use PrairieLearn.

Reviewer: ```suggestion
Next, check out the [instructor guide](../instructor-guide/index.md) for more details on how to use PrairieLearn.
```

# Getting Started

> Welcome to PrairieLearn! You probably want to read / complete the following pages:

Reviewer: ```suggestion
Welcome to PrairieLearn! Check out the following pages to get started:
```

> - [Requesting a course space](../requestCourse/index.md)
> 
>   This page explains how to get access to PrairieLearn.
> 
> - [Fundamental concepts](../concepts/index.md)
> 
>   This page explains at a high level how PrairieLearn works.
> 
> - [Quick start](../getStarted.md)
> 
>   This page explains how to use and navigate the PrairieLearn interface.

Reviewer: ```suggestion
- [Request a course space](../requestCourse/index.md): get access to your own course on PrairieLearn.

- [Fundamental concepts](../concepts/index.md): gain a high-level understanding of how PrairieLearn works.

- [Quick start](../getStarted.md): learn how to use and navigate the PrairieLearn interface.
```
